Dockerfile

```diff
@@ -1,23 +1,36 @@
-FROM openjdk:8-jdk
+FROM maven:3.6.3-jdk-8 as build
 
 ENV JAVA_OPTS "-server -Xconcurrentio -Xmx1g -Xms1g -XX:+UseG1GC -Ddw.server.application_connectors[0].bind_host=0.0.0.0 -Ddw.server.application_connectors[0].port=8989"
 
-RUN mkdir -p /data && mkdir -p /graphhopper
-
 # install node - only required for JS UI
 RUN apt-get install -y wget \
        && curl -sL https://deb.nodesource.com/setup_13.x | bash - \
        && apt-get install -y nodejs
 
-COPY . /graphhopper/
-
 WORKDIR /graphhopper
 
+COPY . .
+
 # create main.js - only required for JS UI
-RUN cd web && npm install && npm run bundleProduction && cd ..
+WORKDIR /graphhopper/web
+
+RUN npm install && npm run bundleProduction
+
+WORKDIR /graphhopper
 
 RUN ./graphhopper.sh build
 
+FROM openjdk:11.0.5-jre
+
+ENV JAVA_OPTS "-server -Xconcurrentio -Xmx1g -Xms1g -XX:+UseG1GC -Ddw.server.application_connectors[0].bind_host=0.0.0.0 -Ddw.server.application_connectors[0].port=8989"
```

Comment:
Can you remove the options -server -Xconcurrentio and -XX:+UseG1GC? We should use the JVM defaults.

Reply:
Sure, I have no clue what all these variables mean so I decided to leave them as they were...

Reply:
Done.